pkgs/development/python-modules/pickledb/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  orjson,
+  aiofiles,
+}:
+
+buildPythonPackage rec {
+  pname = "pickledb";
+  version = "1.3.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "patx";
+    repo = "pickledb";
+    rev = "v${version}";
+    hash = "sha256-19jPKEJ02rRHUYlKs9M0CorDnOMPw2+BpeVQL4Ysj0U=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = [
+    orjson
+    aiofiles
+  ];
+
```

Comment:
Any reason we're not running unit tests? https://github.com/patx/pickledb/blob/master/tests.py